mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -1,6 +1,5 @@
 # Copyright (C) 2023 ISIS Rutherford Appleton Laboratory UKRI
 # SPDX - License - Identifier: GPL-3.0-or-later
-from __future__ import annotations
```

Comment:
Looks like this got lost somewhere in the merges.

Reply:
Thanks for noticing this, will add it back in 👍